Documentation/ldap-connector.md

```diff
@@ -90,6 +90,10 @@ connectors:
     bindDN: uid=seviceaccount,cn=users,dc=example,dc=com
     bindPW: password
 
+    # The attribute to display in the provided password prompt. If unset, defaults
```

Comment:
Just default to `username`? I don't think a lot of the default attributes like `uid` would be very helpful